.ats/main.yaml

```diff
@@ -0,0 +1,5 @@
+app-tests-skip-app-deploy: true
```

Comment:
It seems all the tests are just copied over from linerd2-app repo. Should they be removed here?

Reply:
yeah but I added the multicluster deployment too

Reply:
OK, but then it seems the tests are not executed here.

Reply:
@pipo02mix  Lets keep them in here, we can use it as a starting point